build.gradle

```diff
@@ -73,6 +73,8 @@ dependencies {
     annotationProcessor 'org.projectlombok:lombok:1.18.12'
     compile "org.mapstruct:mapstruct:${mapstructVersion}"
     annotationProcessor "org.mapstruct:mapstruct-processor:1.3.1.Final"
+    compile "org.apache.commons:commons-collections4:4.0"
```

Comment:
Spring framework has apache collection library. If that fulfils our requirement we don't need this dependency.

Reply:
CollectionUtils.emptyIfNull not available in spring framework.